src/test/java/domain/LottosTest.java

```diff
@@ -0,0 +1,35 @@
+package domain;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class LottosTest {
+    private Lottos lottos;
+
+    @BeforeEach
+    void setUp() {
+        List<Lotto> lottoNumbers = new ArrayList<>();
+        lottoNumbers.add(Lotto.of(Arrays.stream(new int[]{1, 2, 3, 4, 5, 6})
+                                    .boxed()
+                                    .collect(Collectors.toList())));
+        lottos = new Lottos(lottoNumbers);
+    }
+
+    @Test
+    void makeStatisticsTest() throws Exception {
+        LottoResult lottoResult = lottos.calculate(Arrays.stream(new int[]{1, 2, 3, 4, 5, 6})
+                                                            .boxed()
+                                                            .collect(Collectors.toList()));
+        LottoPrizeCount lottoPrizeCount = lottoResult.getLottoPrizeCount();
```

Comment:
```suggestion
        LottoResult lottoResult = lottos.calculate(Arrays.asList(1, 2, 3, 4, 5, 6));
```
Arrays.asList() 를 사용해보면 어떨까요? 😄